src/Parser/DemonHunter/Vengeance/Modules/Talents/FeedTheDemon.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage, formatNumber } from 'common/format';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Interface/Others/StatisticBox';
+import SoulFragmentsTracker from '../Features/SoulFragmentsTracker';
+
+//WCL https://www.warcraftlogs.com/reports/ZVJr2MPNx3RCvX6B/#fight=6&source=184
+class FeedTheDemon extends Analyzer {
+  static dependencies = {
+    soulFragmentsTracker: SoulFragmentsTracker,
```

Comment:
I would recommend removing the soulFragmentsTracker dependency and moving all your FTD code into one file.  As best I can tell, the tracker isn't needed to do the FTD analysis but is just a convenient block already setup to fire when soul fragment stacks are removed.  To keep things modular and neat, you should take what you added in the soul fragment tracker and move it in here.  It looks good to go so it should be a quick copy/paste and commit.